tests/integration/cattletest/core/test_container.py

```diff
@@ -364,25 +378,26 @@ def test_container_purge(admin_client, sim_context):
 
     volume = wait_transitioning(admin_client, volume)
     assert volume.state == 'purged'
-
+    container = internal_test_client.reload(container)
+    volume = container.volumes()[0]
     pool_maps = volume.volumeStoragePoolMaps()
     assert len(pool_maps) == 1
     assert pool_maps[0].state == 'removed'
 
 
-def test_start_stop(admin_client, sim_context):
+def test_start_stop(internal_test_client, sim_context):
```

Comment:
I don't immediately see what in this test would require the internal_test_client. Can you verify that admin_client won't suffice?